docs/current_docs/api/return-values.mdx

```diff
@@ -45,9 +45,23 @@ Here is an example of a Dagger Function that returns operating system informatio
 
 Here is an example call for this Dagger Function:
 
+<Tabs groupId="shell">
+<TabItem value="System shell">
+```shell
+dagger -c 'os-info ubuntu:latest'
+```
+</TabItem>
+<TabItem value="Dagger Shell">
+```shell title="Type 'dagger' to start Dagger Shell in interactive mode and enter the command below."
+os-info ubuntu:latest
+```
+</TabItem>
+<TabItem value="Dagger CLI">
```

Comment:
Technically the shell is a part of the CLI. So to contrast with shell is `call` and `core`.

Reply:
> Technically the shell is a part of the CLI. 
I know :) However I couldn't find a good heading for the tab here. I considered `dagger call/core` but when you see it on the page, it's odd that the other tab headings are non-commands and this one is a command. I'm open to suggestions that fit with the style/format of the other tabs, please add them.

Reply:
Ok, let's revisit afterwards then, when there's a better fit. Then we can change everywhere at once.